bin/oref0-setup.sh

```diff
@@ -141,7 +141,8 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
     read -p "What is your pump serial number (numbers only)? " -r
     serial=$REPLY
     echo "Ok, $serial it is."
-    read -p "What kind of CGM are you using? (e.g., G4-upload, G4-local-only, G5, MDT, xdrip?) Note: G4-local-only will NOT upload BGs from a plugged in receiver to Nightscout" -r
+    echo -e "What kind of CGM are you using?\nValid options:\nG4-upload: use Dexcom G4 receiver connected to USB and upload to nightscout\nG4-local-only: will NOT upload BGs from a plugged in receiver to Nightscout\nG5: Dexcom G5\nMDT: Medtronic CGM\nshareble: Bluetooth Dexcom G4 or G5 receiver\nxdrip: Requires xDrip Wireless Bridge (No Receiver)."
+    read -p "Please selection option for your CGM: " -r
```

Comment:
Typo here.  Also would like to get input from folks like @kdisimone and @danamlewis on whether/how we want to change this text.